README.md

```diff
@@ -26,6 +26,90 @@ If the above doesn't help, please [submit an issue](https://github.com/fastlane/
 
 **Note**: If you want to report a regression in _fastlane_ (something that has worked before, but broke with a new release), please mark your issue title as such using `[Regression] Your title here`. This enables us to quickly detect and fix regressions.
 
+## _fastlane_ team
+
+<table>
+<tr>
+<td>
+<img src='https://github.com/lacostej.png?size=200' width=140>
+<h4 align='center'><a href='https://twitter.com/'>Jerome Lacoste</a></h4>
```

Comment:
where did my accents go?

Note that I only added them to see what would happen with them ;)

Reply:
PR? :)

Reply:
@KrauseFx, In the JSON they exist, so the README only needs to be re-generated